docs/config.md

```diff
@@ -674,20 +673,6 @@ Default: `""`
 
 Type: `string` (url)
 
-### `Gateway.FastDirIndexThreshold`
```

Comment:
nit: this header should stay in config docs, at least for a while. Instead, remove only body and mark as removed.
See prior art in https://github.com/ipfs/kubo/blob/master/docs/config.md#discoverymdnsinterval